pom.xml

```diff
@@ -43,13 +43,13 @@
     <distributionManagement>
         <repository>
             <id>releases</id>
-            <name>nukkitx-releases</name>
-            <url>https://repo.nukkitx.com/release</url>
+            <name>opencollab-releases</name>
+            <url>https://repo.opencollab.dev/maven-releases</url>
```

Comment:
We can get rid of this in favour of the solution that we use in the [Nukkit Jenkinsfile](https://github.com/CloudburstMC/Nukkit/blob/12b389a263a0425e057f283d3698d81ff258c69b/Jenkinsfile#L23-L53)